src/models/UiTaskRunCountsByState.ts

```diff
@@ -0,0 +1,5 @@
+import { StateType } from '@/models/StateType'
+
+export type UiTaskRunCountsByState = {
```

Comment:
Could possibly make this more generic as there's nothing specific to task runs here but figured this was fine to start with since no other routes use the backend object ether